src/main/java/com/management/HumanResources/model/EmployeeTime.java

```diff
@@ -1,14 +1,32 @@
 package com.management.HumanResources.model;
 
 import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import org.json.JSONObject;
+
 import lombok.Data;
 
 @Data
+@JsonInclude(JsonInclude.Include.NON_NULL)
 public class EmployeeTime {
 
     private long employeeId;
     private double hoursRemaining;
     private double totalHours;
     private List<TimeOff> timeOffs;
     private String[] availability;
+    private String csvTimeOff;
```

Comment:
I do not think it is a good idea to have these two CSV fields here. Do you think it would be a better idea to create two methods named `setTimeOffFromCsv` and `setAvailabilityFromCsv` that would call the respective setters?

Reply:
In relation to our database and client these strings will never be ever seen. The `@JsonInclude` will remove these automatically whenever we want to read TimeOff values.